### DISCUSSION TITLE
How to create own BBCode and pass unique attributes on Vue Component

### DISCUSSION THREAD

**Discussion by narendra087** [JiLiZART/BBob#109] [Q&A] [answered]
Hi,
can you help me how to create own BBCode and pass unique attributes on vue component.
Example BBCode:
`[QUOTE=Narendra;25/06/2021;id001]Quote Content[/QUOTE]`
to this template component:

> <template>
    <div class="q-wrap">
        <div class="q-chat">
            <p class="q-header">25/06/2021, Narendra said:</p>
            <hr>
            <slot></slot>
        </div>
    </div>
</template>

Thank you.

**Comment by JiLiZART** [comment, marked as answer]
All params from bbcodes passed as DOM attrs (soon I update this behavior to pass as props too)

I created sandbox with example https://codesandbox.io/s/cocky-banzai-q8zs7?file=/src/App.vue
